dcpy/connectors/edm/product_metadata.py

```diff
@@ -21,3 +22,11 @@ def fetch_raw(product: str, *, dataset: str | None = None) -> str:
 
 def fetch(product: str, *, dataset: str | None = None) -> md.Metadata:
     return md.Metadata(**yaml.safe_load(fetch_raw(product, dataset=dataset)))
+
+
+def download(product: str, output_path: Path, *, dataset: str | None = None) -> Path:
+    if not output_path.parent.exists():
```

Comment:
just fyi, you can one-line this with `mkdir(parents=True, exists_ok=True)`

Reply:
done!